packages/lisk-transaction-pool/test/transaction_pool.ts

```diff
@@ -181,4 +189,20 @@ describe('transaction pool', () => {
 			);
 		});
 	});
-});
+
+	describe('#expireTransactions', () => {
+		let removeTransactionsFromQueuesStub: sinon.SinonStub;
+		let expireTransactions: () => Promise<ReadonlyArray<Transaction>>;
+
+		beforeEach(() => {
+			removeTransactionsFromQueuesStub = sandbox.stub(transactionPool as any, 'removeTransactionsFromQueues');
```

Comment:
do we need `as any` here?

Reply:
yes, because `removeTransactionsFromQueues` is a private function.